packages/amplify_authenticator/lib/src/widgets/buttons.dart

```diff
@@ -102,16 +113,18 @@ class GoToSignUpButton extends StatelessWidget {
   Widget build(BuildContext context) {
     final SignInViewModel _authModelView =
         InheritedAuthViewModel.of(context)!.signInViewModel;
+    final NavigationResolver _strings =
```

Comment:
Nit: It seems more appropriate to name this `_navigationResolver` or `_textResolver` or `_stringResolver` or `_resolver`. Semantically, `_strings` implies `List<String>` imo.

Reply:
That's fair. Fixing this.